packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -136,10 +137,23 @@ export const httpMiddleware = ({ dispatch }) => next => (action) => {
 				stack: error.stack,
 			},
 		}, action);
-		dispatch(httpError(newAction.error));
-		if (newAction.onError) {
-			dispatch(onError(newAction, newAction.error));
-		}
+
+		// clone the response object else the next call to text or json
+		// triggers an exception Already use
+		newAction.error.stack.response.clone().text().then((response) => {
+			try {
+				newAction.error.stack.response = response;
+				newAction.error.stack.messageObject = JSON.parse(response);
+			} finally {
+				dispatch(httpError(newAction.error));
+				if (newAction.onError) {
+					const onErrorAction = onError(newAction, newAction.error);
+					if (onErrorAction) {
+						dispatch(onError(newAction, newAction.error));
+					}
```

Comment:
if the function exists it's an action creator so it must return an action object.
here you fail silently. please remove this if and just call the dispatch on it
dispatch(onError(newAction, newAction.error));